public/less/themes/arc-green.less

```diff
@@ -85,6 +85,47 @@ body {
     color: #9e9e9e;
 }
 
+/* firefox scroll bars */
+
+* {
+    scrollbar-width: thin;
+    scrollbar-color: #87ab63 rgba(255,255,255,0.1);
+}
+
+/* webkit scrollbars */
+
+body ::-webkit-scrollbar,
```

Comment:
Any reason to keep those `body ::-webkit-scrollbar`? I'd remove them as well. Otherwise LGTM.

Reply:
It seems that it is needed to not be overriden by semantic. https://github.com/go-gitea/gitea/pull/7269#discussion_r297452378

Reply:
Yeah its overwritten by semantic otherwise.
I could also use `!important` but i don't like this flag

Reply:
As mentioned in the other resolved comment, I'd prefer the use of `!important` to outrule other styles. That way, the intention is much more clear than with arbitrary prefixes.

Reply:
Overruling conflicting semantic styles with `!important` is totally fine imho. We do it in quite a few places actually.

Reply:
then i will change to !important

Reply:
done